te_markdown/heroes-create-testcase.md

```diff
@@ -67,7 +67,7 @@ The test case is expected to be written by a tester before the application is in
 
 * It represents a concrete test execution plan that implements a test specification and can be readily understood by the domain expert who wrote the specification
 * It is concrete insofar as to be executable 
-* It is the concrete and detailed expection developers can realize their application for
+* It is the concrete and detailed expectation developers can realize their application for
 * It functions as a bridge between actual feature implementation (developer) and feature specification (domain expert), working as an artifact for discussion between domain expert, tester and developer
 
 It is quite common to have the test case change because of implementation issues that could not have been foreseen. Changes need to be discussed though, which again has the benefit of sharing a common understanding of the feature to be implemented between domain expert, tester and developer. 
```

Comment:
```suggestion
It is quite common to have the test case change because of implementation issues that could not have been foreseen. Changes need to be discussed though, which again has the benefit of sharing a common understanding of the feature to be implemented, between domain expert, tester, and developer. 
```